hack/k8s/helm/nuclio/Chart.yaml

```diff
@@ -15,7 +15,7 @@
 apiVersion: v1
 description: Serverless for Real-Time and Data-Driven Applications
 name: nuclio
-version: 0.20.1
```

Comment:
No need to change the chart version manually, as the release flow handles it.
Updating it manually diverges it and can possibly cause issues.